systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -253,4 +254,34 @@ public void testClusterWithCustomParameters() {
         assertThat(jsonString, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(jsonString, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
+    public void testDeployKafkaOnPersistentStorage() {
+        int expectedZKPods = 3;
+        int expectedKafkaPods = 3;
+        Oc oc = (Oc) this.kubeClient;
+        String clusterName = "openshift-my-cluster";
+        oc.newApp("strimzi-persistent", map("CLUSTER_NAME", clusterName));
```

Comment:
Have you considered setting the `ZOOKEEPER_NODE_COUNT` and `KAFKA_NODE_COUNT` template parameters (or if you write the test using a CM as suggested by @scholzj, the `zookeeper-nodes` and `kafka-nodes` CM keys). Both default to 3, which means you spend time waiting for all 3 pods to be set up by the SS. You could set the both to 2 and still be covering the "PVC with multiple pods" case, but the test should run slightly quicker.

Reply:
@tombentley It's a good point. Changed values to 2